src/Reservation.tsx

```diff
@@ -391,7 +415,7 @@ export default function Resevation(): JSX.Element {
                       id="startTime"
                       label="Start Time"
                       type="time"
-                      defaultValue={selectedStartDate}
+                      value={formatTime(roundToMinuteInterval(selectedStartDate!, 15))}
```

Comment:
Is the `!` needed here?

Reply:
I added `!` since `roundToMinuteInterval` expects a Date while `selectedStartDate` is either a Date or null - and since `selectedStartDate` is first set to the current time I believe it'd be safe to assume that `selectedStartDate` is never null.

Reply:
Could we get rid of the ` | null` when we define it instead?